### PR TITLE
Add Green Difference Vegetation Index

### PR DESCRIPTION
**Pull request by j-miszczyszyn**
Hello, second try after little correction

### CODE REVIEW

File: src/indices.py [hunk 1]
```diff
@@ -432,8 +432,8 @@ class SpectralIndices(BaseModel):
             date_of_addition="2021-05-13",
             contributor="https://github.com/davemlz",
         ),
-        GDVI=SpectralIndex(
```
Comment (davemlz) [inline src/indices.py:435]: Please do not remove existing keys from indices in the list.

File: src/indices.py [hunk 2]
```diff
@@ -432,8 +432,8 @@ class SpectralIndices(BaseModel):
...
-            short_name="GDVI",
```
Comment (davemlz) [inline src/indices.py:436]: Please do not remove existing attributes from indices in the list.

File: src/indices.py [hunk 3]
```diff
@@ -2106,5 +2106,14 @@ class SpectralIndices(BaseModel):
             date_of_addition="2023-07-03",
             contributor="https://github.com/geoSanjeeb",
         ),
+        GDVI=SpectralIndex(
```
Comment (davemlz) [inline src/indices.py:2109]: The key "GDVI" is already existing for the Generalized Difference Vegetation Index (GDVI), please update your key to a unique key.

File: src/indices.py [hunk 4]
```diff
@@ -2106,5 +2106,14 @@ class SpectralIndices(BaseModel):
...
+    short_name='GDVI',
```
Comment (davemlz) [inline src/indices.py:2110]: Please update the `short_name` with the key of the index.

File: src/indices.py [hunk 5]
```diff
@@ -2106,5 +2106,14 @@ class SpectralIndices(BaseModel):
...
+    long_name='Generalized Difference Vegetation Index ',
```
Comment (davemlz) [inline src/indices.py:2111]: This name is from another index. Please update the name with the actual index name.

File: src/indices.py [hunk 6]
```diff
@@ -2106,5 +2106,14 @@ class SpectralIndices(BaseModel):
...
+    formula='N-G',
+    reference='https://doi.org/10.1016/0034-4257(79)90004-X',
```
Comment (davemlz) [inline src/indices.py:2113]: This paper does not correspond to the formula presented in the `formula` attribute. Please update the reference doi with the actual source paper of the index.

**Review by davemlz — CHANGES_REQUESTED** [review]
Hi @j-miszczyszyn,

I have reviewed the index and before acceptance there are a few changes that need to be made.

Please let me know when everything is corrected for adding the index!

Cheers!

File: src/indices.py (hunk 1 above)
Comment (j-miszczyszyn) [reply src/indices.py:435]: I did not remove the existing index

File: src/indices.py (hunk 2 above)
Comment (j-miszczyszyn) [reply src/indices.py:436]: I did not remove the existing index

**Comment by j-miszczyszyn** [comment]
Now i see what should I do, but i have the ask. 
What should I call the new index in order not to mislead anyone? I am not the author of tje publication and only used it in research. The article cited is from 1979 and is older than the one listed (2013)

File: src/indices.py (hunk 1 above)
Comment (davemlz) [reply src/indices.py:435]: No, but the key was removed: https://github.com/j-miszczyszyn/awesome-spectral-indices/blob/76b8ca5399391b039c12ad38c9fc822e53be3962/src/indices.py#L435

File: src/indices.py (hunk 2 above)
Comment (davemlz) [reply src/indices.py:436]: No, but the `short_name` attribute was removed: https://github.com/j-miszczyszyn/awesome-spectral-indices/blob/76b8ca5399391b039c12ad38c9fc822e53be3962/src/indices.py#L436

**Comment by j-miszczyszyn** [comment]
Hey,
I checked more publications and understood what the differences are. In the literature we come across GDVI with two patterns, different names but the same abbreviation.

Please take a look at this article DOI: 10.31914/aak.2264
It describes the difference.
obraz

Another article also uses the Trucker formula https://link.springer.com/article/10.1007/s40808-015-0029-y

We have been faced with the daunting task of sorting out the indicators. It therefore proposes the index "GDVI_Trucker" to clearly communicate the differences from the GDVI developed by Wu. What do you think?

**Pull request closed without merging**